app/controllers/cases/offender_sar_complaint_controller.rb

```diff
@@ -0,0 +1,53 @@
+module Cases
+  class OffenderSarComplaintController < OffenderSarController
+    include OffenderSARComplaintCasesParams
+
+    def initialize
```

Comment:
May be more Railsy to use a before filter here rather than `initialize`:

https://stackoverflow.com/questions/18576150/ruby-on-rails-before-filter-vs-rubys-initialize

Reply:
That was an interesting rabbit hole... 😁 It turns out you need to call `super` within the initialize method to make the view render inside the layout properly - not sure why! But all the correspondence types follow this pattern. You can omit the other instance vars in the initialize method and the new complaint tests still pass, but I've left it as-is for consistency with all the other controllers for now, maybe one day we'll find out why it's necessary.